docs/host/server_setup/linux-server.md

```diff
@@ -0,0 +1,132 @@
+# Host a Domain from a Local or Cloud Linux Server
+
+
+The Vircadia packages can help you get your own domain up and running quickly.
+
+## Installation
+
+You can run these same commands on an existing Vircadia domain to upgrade it if the original domain was installed using the package. Packages are currently available for the following distributions:
+
+### Ubuntu Server 18.04 LTS (2022.1.1 Selene)
+
+```sh
+wget https://github.com/vircadia/vircadia/releases/download/2022.1.1/vircadia-server-2022.1.1-Selene-Ubuntu18.04_amd64.deb
+sudo apt-get update && sudo apt-get install ./vircadia-server-2022.1.1-Selene-Ubuntu18.04_amd64.deb
+```
+
+### Ubuntu Server 20.04 LTS (2022.1.1 Selene)
+
+```sh
+wget https://github.com/vircadia/vircadia/releases/download/2022.1.1/vircadia-server-2022.1.1-Selene-Ubuntu20.04.4_amd64.deb
+sudo apt-get update && sudo apt-get install ./vircadia-server-2022.1.1-Selene-Ubuntu20.04.4_amd64.deb
+```
+
+### Rocky Linux 8.x (2022.1.1 Selene)
+
+```sh
+sudo yum install https://github.com/vircadia/vircadia/releases/download/2022.1.1/vircadia-server-2022.1.1-Selene.Rocky8.x_x86_64.rpm
+```
+
+### Unlisted Distribution
+
+If you do not see your distribution listed here, you may compile your own server from source with the [Linux build guide](https://github.com/vircadia/vircadia/blob/master/BUILD_LINUX.md).
+
+## Configuration
+
+The installation packages will create a domain at the default port location and configure a service to keep it running on that machine.
+
+For the list of network ports that you will need to open and manage, see [here](../configure-settings/network-settings).
+
+Connect a web browser to the server at port 40100. (If you are on the machine that the server is running on, this would be http://localhost:40100) Complete the initial setup wizard and you should have a functioning domain.
+
+## Files and Server Configuration
+
+The program files are installed in /opt/vircadia:
+ - **/opt/vircadia** contains the executables
```

Comment:
This gets seen as an email, find out how to prevent Docusaurus from detecting these as email addresses.